stacks-signer/src/v0/signer.rs

```diff
@@ -372,6 +405,7 @@ impl Signer {
             "burn_height" => block_proposal.burn_height,
         );
         crate::monitoring::increment_block_proposals_received();
+        #[allow(unused_mut)]
```

Comment:
I see why you did this, but it looks a little wrong. Would this be more clear?
```rust
        #[cfg(any(test, feature = "testing"))]
        let mut block_info = BlockInfo::from(block_proposal.clone());
        #[cfg(not(any(test, feature = "testing")))]
        let block_info = BlockInfo::from(block_proposal.clone());
```
I don't have a strong opinion on this, just a suggestion.

Reply:
Sure :)